lib/routes/authorization.js

```diff
@@ -19,6 +18,13 @@ const verify = require('../browserid');
 const CODE = 'code';
 const TOKEN = 'token';
 
+const UNTRUSTED_CLIENT_SCOPES = [
+  'profile:uid',
+  'profile:email',
+  'profile:avatar',
```

Comment:
"By default"? At least in the context of this PR, until a permissions page is built into the UI, any scope in this list is plain not allowed, default or otherwise.

Does that mean we don't Pocket an avatar until we've added a permissions page?